libdevcore/CommonData.cpp

```diff
@@ -80,7 +80,11 @@ int dev::fromHex(char _i)
 
 bytes dev::fromHex(std::string const& _s, ThrowType _throw)
 {
-	unsigned s = (_s[0] == '0' && _s[1] == 'x') ? 2 : 0;
+	bool isPrefix = (_s[0] == '0' && _s[1] == 'x');
```

Comment:
this is temporary changes. 
the idea is to make fillers HEX fields starting with "0x.."
then it would be easier to make every test in Test repository to satisfy this condition.

1) each "data" starting with 0x
2) each "to", address field starting with 0x
3) each hash field prefixed 0x 

as I understand other clients are having issues when reading .json test files hex string lines